src/components/inputs/default-input/index.jsx

```diff
@@ -0,0 +1,43 @@
+import { InputAdornment } from "@mui/material";
+import StyledTextField from "../styles";
+import { TextField } from "@mui/material";
+
+const DefaultInput = ({
+  adorment = null,
+  placeholder = "place holder",
+  isAdormentStart = true,
+  text = "",
+  type,
+  size,
+  handler,
+}) => {
+  // {type} prop can be: "filled", "outlined", or "standard"
+  // {isStart} prop is a boolen that shows if the adorment displays at start or end of input
```

Comment:
is this comment needed? 
I can't see isStart Prop

Reply:
Yes, it's the same as "isAdormentStart", I changed the name in the comment.